.github/workflows/test-updatecli-run.yml

```diff
@@ -21,9 +21,43 @@ permissions:
 
 jobs:
   test:
+    needs:
+      - version
+      - version-file
+      - default
+    runs-on: ubuntu-latest
+    steps:
+      - id: check
+        uses: elastic/oblt-actions/check-dependent-jobs@v1
+        with:
+          jobs: ${{ toJSON(needs) }}
+      - run: ${{ steps.check.outputs.is-success }}
+
+  default:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./updatecli/run
+        with:
+          command: help
+
+  version:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./updatecli/run
+        with:
+          command: help
+          version: v0.86.0
+
+  version-file:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - name: Setup version file
+        run: |
+          echo "v0.87.0" > .tools-version
```

Comment:
is this correct?

should it be something like:

```
updatecli v0.87.0
```

?

Reply:
It supports both cases:
- see https://github.com/updatecli/updatecli-action/pull/828/files#diff-27653c212e1cfe533e4eb2f7d0d3f89604c9de48a09583b4cbbbcbd08a07da79R139

Reply:
However, I've just added support for both cases:
- [`ef7d814` (#176)](https://github.com/elastic/oblt-actions/pull/176/commits/ef7d814deadfba9fd5e939152979032fa8bb0cbe)